platform/security/handler/security-handler-pki/src/main/java/org/codice/ddf/security/handler/pki/CrlChecker.java

```diff
@@ -50,16 +50,13 @@
     private static String encryptionPropertiesLocation = new AbsolutePathResolver(
             "etc/ws-security/server/encryption.properties").getPath();
 
-    private AtomicReference<CRL> crlCache = new AtomicReference<>();
+    private static AtomicReference<CRL> crlCache = new AtomicReference<>();
```

Comment:
✏️ Nit, looks like this could be final as well? At least I don't see it being reassigned anywhere else.

Reply:
That's a good point. It could have been final before when it was a member field as well. Missed it. But I think I'm going to let this slide for now so we don't have to run CI again, with the thinking that this should be re-written as a service.

Reply:
I'm ok with that reasoning. 👍